src/vs/workbench/contrib/notebook/common/model/notebookTextModel.ts

```diff
@@ -402,7 +402,7 @@ export class NotebookTextModel extends Disposable implements INotebookTextModel
 			true,
 			undefined, () => undefined,
 			undefined,
-			true
```

Comment:
this was strange and feels like it should be false. It's only called on creation of notebooks as far as I could tell: Create a notebook, ctrl+z - a no-op undo is performed